deploy/template.yaml

```diff
@@ -86,6 +86,7 @@ Mappings:
       contactUrl: "https://home.build.account.gov.uk/contact-gov-uk-one-login"
       templateCaching: true
       dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+      spotAccountNumber: "arn:aws:iam::738810260032:root"
```

Comment:
May be more accurate to call this `spotAccountArn` as it's an ARN rather than a number. Also, the rest of the file does not add quotes around ARNs.

Reply:
Number -> Arn